test/fixtures/forks/zcash/README.md

```diff
@@ -0,0 +1,126 @@
+# Generate test vectors for new ZCash testnet consensus branch IDs
```

Comment:
Writing the procedure out here is great! :+1: 

Nit regarding the commit message of da9d22b0 (Add new test vector for tZEC Canopy P2SH 2-of-3), the "Ticket: BG-24753" must be the last line. IIRC, some Jira automation relies on the order here.

Reply:
ah thanks, updated in 7434dc4b5b25c7939f409ba8f64c079f3d862aa4